pkg/newstorage/newstorage.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package newstorage
+
+import "errors"
+
+// ErrStoreNotFound is returned when a store is not found.
+var ErrStoreNotFound = errors.New("store not found")
+
+// ErrDataNotFound is returned when data is not found.
+var ErrDataNotFound = errors.New("data not found")
+
+// StoreConfiguration represents the configuration of a store.
+type StoreConfiguration struct {
+	TagNames []string
```

Comment:
It would be good to comment on the relationship of this property with the Tag struct on L40.

Reply:
Done in my new PR: https://github.com/hyperledger/aries-framework-go/pull/2426/files#diff-98db716f6d6bedd511a5f903dc6d785b6f60df130787f50612d64804a1f8195fR46